### ISSUE TITLE
Add setting option for download location

### ISSUE THREAD

**Issue by Theblackcat98** [dstark5/Openlib#78]
As the title states, I think it would be a welcome addition to be able to change the default download location for books.

**Comment by Asmageddon** [comment]
Yeah, this looks much better as a frontend/downloader than as a reader tbh.

**Comment by dstark5** [comment]
Yeah sure will add it in the future updates

**Comment by shuvashish76** [comment]
Duplicate of #32 isn't it? Close one of the issue.
